sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -537,6 +543,14 @@ export class ApplicationStudentsController extends BaseController {
         );
       appealStatus = mostRecentAppeal?.status;
     }
+    let mostRecentApplicationOfferingChangeRequestStatus: ApplicationOfferingChangeRequestStatus;
+    if (application.applicationStatus === ApplicationStatus.Completed) {
+      mostRecentApplicationOfferingChangeRequestStatus =
+        await this.applicationOfferingChangeRequestService.getApplicationOfferingChangeRequestStatus(
```

Comment:
I believe, I asked you in our call. were these details not be able to retrieve from line 526 service??